chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -674,7 +699,12 @@ impl PeerManagerActor {
                 let _guard = run_later_span.enter();
                 // Start syncing network point of view. Wait until both parties are connected before start
                 // sending messages.
-                let known_edges = act.network_graph.read().edges().values().cloned().collect();
+                let mut known_edges: Vec<Edge> =
+                    act.network_graph.read().edges().values().cloned().collect();
+                if act.config.skip_sending_tombstones {
+                    known_edges.retain(|edge| edge.removal_info().is_none());
+                    metrics::EDGE_TOMBSTONE_SENDING_SKIPPED.inc();
+                }
```

Comment:
So here we are *completely* skipping tombstones, right? So the new node will effectively *never* learn about tombstones (and not only during the first two minutes). Is this fine? Perhaps it's better to not skip tombstones altogether, but rather send them after some delay in several messages?

Reply:
Yes - that's why code is technically incorrect. The problem is that we have FAR too many tombstones in shardnet - and until we have 'time-based' nonces, we don't know how old they are, and which ones are safe to remove.

That's why I'm planning to enable this only in shardnet - as a temporary measure.